openwisp_notifications/base/models.py

```diff
@@ -250,3 +251,15 @@ def web_notification(self):
         if self.web is not None:
             return self.web
         return self.type_config.get('web_notification')
+
+
+class AbstractObjectNotification(UUIDModel):
```

Comment:
from reading this code is not immediately clear what this model does.

Does it serve as ignore?
Does every object need this? Or is it added only when a user deactivated notifications for a particular object?